hardhat.config.js

```diff
@@ -8,7 +8,19 @@ module.exports = {
       accounts: {
         mnemonic: 'test test test test test test test test test test test junk'
       }
-    }
+    },
+    // Add this network to your config!
+    optimism: {
```

Comment:
Shouldn't be included in the initial config

Reply:
I included it so the team could help me debug 😔